### ISSUE TITLE
Offset ... hasn't been loaded yet

### ISSUE THREAD

**Issue by EnWo** [aadsm/jsmediatags#58]
Hello,

I've experienced a similar behaviour as the following issues:

[https://github.com/aadsm/jsmediatags/issues/29](url)
[https://github.com/aadsm/jsmediatags/issues/49](url)

Object {type: "parseData", info: "Offset 315 hasn't been loaded yet."}
[04 - Turn! Turn! Turn! (To Everything There Is A Season).zip](https://github.com/aadsm/jsmediatags/files/1131139/04.-.Turn.Turn.Turn.To.Everything.There.Is.A.Season.zip)

or

Object {type: "parseData", info: "Offset 274 hasn't been loaded yet."}
[01 - Volunteers.zip](https://github.com/aadsm/jsmediatags/files/1131144/01.-.Volunteers.zip)

For my use case I have fixed the files by re-writing the ID3 tags with winamp.
Maybe the files are helpful to debug the offset problems.

Thanks,
Enno

**Comment by aadsm** [comment]
I found the issue, the ID3 frames in this mp3 file are indeed wrongly formatted.

For some reason the software used to generate them (MP3ext) inserts data at the end of the ID3 tag in a format that is not specified by the spec (it's not even a frame and the ID3 tag should be a collection of frames). It inserts the string "MP3ext V3.3.19(ansi)" multiple times after the last frame breaking the read logic.

![screen shot 2017-07-08 at 9 04 02 am](https://user-images.githubusercontent.com/78122/27987032-7507f69c-63bc-11e7-9de4-55294736a40e.png)

I've searched around and found other ID3 reading software that has to explicitly accommodate for this error created by MP3ext. Apparently it's a well known problem:

>  MP3ext V3.3.17 places a non-compliant padding string at the end of the ID3v2 header. This is supposedly fixed in v3.4b21 but only if you manually add a registry key. This fix is not yet confirmed.

I can explicitly look out for this pattern and consider it as the end of the ID3 tag.
